.gitignore

```diff
@@ -279,6 +279,15 @@ tests/unit/unit_tests
 tests/unit/punit_tests
 tests/unit/sedov_tests_*
 tests/unit/psedov_tests_*
+tests/unit/tmop_tests_*
+tests/unit/ptmop_tests_*
+tests/unit/cube.mesh
+tests/unit/star.mesh
+tests/unit/blade.mesh
+tests/unit/square01.mesh
+tests/unit/toroid-hex.mesh
+tests/unit/beam-hex-nurbs.mesh
+tests/unit/square-disc-nurbs.mesh
```

Comment:
I'd prefer to avoid copying these mesh files. In other places in the tests we just use full relative names: e.g. https://github.com/mfem/mfem/blob/bb2a80ed0af870a3f1a861d94f3d4fc6561bb663/tests/unit/fem/test_assembly_levels.cpp#L121

Reply:
do we have also access to `../../miniapps/meshing`?

Reply:
We are already copying the 'data' to the build directory with cmake

Reply:
`done`

Reply:
You can now remove them from `.gitignore` too.

Reply:
hum, the tests are failing, we need somehow to copy them before the runs - because of `../../miniapps/meshing`

Reply:
Hmm, yes, I guess for the out-of-source builds (cmake and gmake) the mesh files in `miniapps/meshing` are not copied to the build location. We can add rules for that in `miniapps/meshing/{CMakeLists.txt,makefile}` and invoke them when building the tmop tests. This should also help with running the sample runs in `miniapps/meshing` with out-of-source builds.

Reply:
This issue should be resolved now.